core/session.ts

```diff
@@ -444,6 +431,21 @@ export class Session {
     return encode64(dataform.CompiledGraph, this.compile());
   }
 
+  public normalizeDatabase(database: string): string {
+    return this.adapter().normalizeIdentifier(
+      `${database}${this.getDatabaseSuffixWithUnderscore()}`);
+  }
+
+  public normalizeSchema(schema: string): string {
+    return this.adapter().normalizeIdentifier(
+      `${schema}${this.getSchemaSuffixWithUnderscore()}`);
+  }
+
+  public normalizeTable(table: string): string {
```

Comment:
nit: i think "normalize" isn't quite the right word to use here. elsewhere in the code, "normalizing" an identifier means: to translate (usually involves escaping, casing, whatever) this string to something which can be used as an identifier in the target warehouse.

here we *are* normalizing, sure, but we're doing more than that - we are suffixing/prefixing as necessary.

i would suggest "finalize", i.e. `finalizeDatabase`

Reply:
Sounds good, done - thanks.